scipy/stats/tests/test_continuous_basic.py

```diff
@@ -120,7 +124,8 @@ def test_cont_basic(distname, arg):
     check_sf_isf(distfn, arg, distname)
     check_pdf(distfn, arg, distname)
     check_pdf_logpdf(distfn, arg, distname)
-    check_pdf_logpdf_at_endpoints(distfn, arg, distname)
+    if distname != 'boost.beta':  # TODO: figure out why boost.beta not working
```

Comment:
I can't say why it's not working, but when `boost.beta.pdf` is evaluated with args `(2.3098496451481823, 0.6268795430096368)` at `x=1`, it is crashing the console.

Reply:
From [wikipedia](https://en.wikipedia.org/wiki/Beta_distribution): "... the inclusion of x = 0 and x = 1 does not work for α, β < 1".  It turns out for our case that Boost handles x=0, but it's hitting an overflow error in the `ibeta_derivative` function when x=1 and β<1.  This might be a bug in the implementation, I'm looking into it now

Reply:
I'm not sure how fast Boost devs will be. In the meantime, could you just have the wrapper look for `x>=1` and return 0?

Reply:
It looks like PR https://github.com/boostorg/math/pull/460 into the boost develop branch going through CI right now, not sure when it will be released though.  I had it returning inf, but 0 is the correct answer here

Reply:
I think my comment was wrong here. I think the correct answer is inf.